config/test/tsconfig.json

```diff
@@ -1,25 +1,34 @@
 {
   "compilerOptions": {
-    "target": "ES5",
+    "target": "es5",
     "jsx": "react",
     "module": "commonjs",
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "declaration": true,
+    "declaration": false,
```

Comment:
Why did we change this?

Reply:
The compiled test files do not need TypeScript declaration (.d.ts) files (this is not the same as a sourcemap for debugging)